scripts/create-env.js

```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+
+fs.writeFileSync('./.env', `API_KEY=${process.env.API_KEY}\n`);
+fs.writeFileSync('./.env', `APP_ID=${process.env.APP_ID}\n`);
```

Comment:
- [ ] **Optional:** These keys should be ignored in a more realistic development setting. They can compromise the security of your app very quickly. 😠